extra/1-radio-stations.js

```diff
@@ -14,7 +14,13 @@
  */
 
 // `getAllFrequencies` goes here
-
```

Comment:
Hi, Mortaza, here is my comment on your radio function. well done to initialize an empty array then using the loop to fill it with frequencies starting from i= 87 to i= 108.great